src/apps/api/views/mapping.py

```diff
@@ -1,83 +1,102 @@
 from django.http import HttpResponse
-from src.apps.api.utils import finish_request
 
 import bson.json_util as jsonb
-import common.schemas.utils.data_utils as d_utils
 import json
 
-def get_standard_data(standard_list=STANDARDS, manifest_type=str(), queryByManifestType=False):
-    with open(STANDARDS_MAPPING_FILE_PATH) as f:
-        standards_file_data = json.load(f)
-        standards_file_data = standards_file_data[0]
-        output = list()
-        output_dict = dict()
-
-        if standard_list and not queryByManifestType:
-            for tol_column, standard_data in standards_file_data.items():
-                tol_column_dict = dict()
-                tol_column_dict[tol_column] = dict()
-                
-                for standard in standard_list:
-                    if standard_data.get(standard, str()):
-                        tol_column_dict[tol_column] |= {standard: {'key': standard_data.get(standard,str()).get('field', str())}}
-                            
-                        # Merge dictionaries
-                        output_dict |= tol_column_dict
-                            
-            output.append(output_dict)
-
-            result = finish_request(template=output, error=None, num_found=len(output[0]), return_http_response=False)
 
-        elif any(x in standard_list for x in STANDARDS) and queryByManifestType and manifest_type:
-            manifest_type_columns = DTOL_EXPORT_TO_STS_FIELDS.get(manifest_type, str())
+def get_mapped_fields_for_project(standard, project):
+    if standard not in STANDARDS:
+        return None
 
-            for tol_column, standard_data in standards_file_data.items():
-                tol_column_dict = dict()
-                tol_column_dict[tol_column] = dict()
+    with open(settings.STANDARDS_MAP_FILE_PATH) as f:
+        standard_mapped_data = json.load(f)
+        
+        if standard and not project:
+            output_list = list()
 
-                for standard in standard_list:
-                    if standard_data.get(standard, str()) and tol_column in manifest_type_columns:
-                        tol_column_dict[tol_column] |= {standard: {'key': standard_data.get(standard,str()).get('field', str())}}
-                        
-                        # Merge dictionaries
-                        output_dict |= tol_column_dict
+            for tol_field, mapped_data in standard_mapped_data.items():
+                if standard != 'tol':
+                    field = mapped_data.get(standard, dict()).get('field', str())
+                else:
+                    field = tol_field
+                
+                # Ignore if 'field' is nonexistent
+                if field:
+                    output_list.append(field)
+
+            output_list.sort()
+            return output_list
+        
+        elif standard and project:
+            output_list = list()
+            project_fields = DTOL_EXPORT_TO_STS_FIELDS.get(project, str())
```

Comment:
As we expect "project_fields" is a list, change it to:

`project_fields = DTOL_EXPORT_TO_STS_FIELDS.get(project, list())`

It seems we didn't update DTOL_EXPORT_TO_STS_FIELDS even there is no column in the manifest.  Just thinking whether we can just export all the project column based on the schema -> specification.  Need to check it with @shaw2thefloor